lib/vsc/mympirun/pmi/sched.py

```diff
@@ -53,8 +53,9 @@ def __init__(self, nodes=None, cores=None, ranks=None, mem=None, gpus=None):
 
     def __str__(self):
         """Human readable"""
-        return "{} nodes; with per node {} cores, {} ranks, {} mem, {} gpus".format(
-            self.nodes, self.cores, self.ranks, self.mem, self.gpus)
+        res = f"{self.nodes} nodes; with per node {self.cores} cores, {self.ranks} ranks, "
+        res += "{self.mem} mem, {self.gpus} gpus"
+        return res
```

Comment:
you don't need to have a variable to split the lines. you can use parentheses to split a long string over more than 1 line. it will still print a single line:
```
return (f" my first part of {self.string}"
    "my second part")
```
for example:
```
>>> x='test'
>>> print((f"allo {x} more_test"
... "allo")
... )
allo test more_testallo
```